src/extensions/lens-renderer-extension.ts

```diff
@@ -60,4 +62,27 @@ export class LensRendererExtension extends LensExtension {
   async isEnabledForCluster(cluster: Cluster): Promise<Boolean> {
     return (void cluster) || true;
   }
+
+  listenIpc<
```

Comment:
Could we move all extension "ipc" methods to separate (singleton?) class? If we bundle these methods to extension class itself it means that extension needs to pass extension instance around just to be able to communicate via ipc.. which feels a bit clumsy.